<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyTacToeGame</class>
 <widget class="QWidget" name="PyTacToeGame">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyTacToe</string>
  </property>
  <widget class="QPushButton" name="gameSquare00">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare01">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare02">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare10">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare11">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare12">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare20">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare21">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameSquare22">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>120</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerOneNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>You</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="gameStartStopButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>420</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playerSwap">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Swap Players</string>
   </property>
   <property name="text">
    <string>⬌</string>
   </property>
  </widget>
  <widget class="QComboBox" name="enemySelect">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Select Enemy Level</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>  Medium</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>  Easy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>  Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>  Hard</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
